templates/types/streaming/nextjs/app/api/chat/config/llamacloud/route.ts

```diff
@@ -5,6 +5,9 @@ import { LLamaCloudFileService } from "../../llamaindex/streaming/service";
  * This API is to get config from the backend envs and expose them to the frontend
  */
 export async function GET() {
+  if (!process.env.LLAMA_CLOUD_API_KEY) {
+    return NextResponse.json({ status: 404 });
```

Comment:
how about setting an error? (not sure we can do that with 404):
```suggestion
    return NextResponse.json({ 
    error: "env variable LLAMA_CLOUD_API_KEY is required to use LlamaCloud"
    status: 404 });
```